ansible/roles/delayed_job/tasks/main.yml

```diff
@@ -1,12 +1,18 @@
 ---
   - set_fact: delayed_job_installed="true"
 
+  - name: Monit Stop All
+    shell: monit stop all
+    become: true
+
   - name: Install delayed_job monit script
     template:
       src: delayed-job-monit-rc
       dest: /etc/monit/conf.d/delayed_job_{{project_name}}_{{rails_env}}
     become: true
-    notify: reload_monit
+    notify:
+      - restart_monit
```

Comment:
Do you want restart or reload first? Lower down in the diff, there's a task with reload first then restart.

Reply:
I'll reorder them but the reason i changed the order the handlers are defined is because thats the order it actually runs them in apparently: https://selivan.github.io/2017/01/15/ansible-multiple-handlers-subtlety.html